### PR TITLE
vlan member tagging / untagging issue #38

### LINKED ISSUE #38: vlan member tagging / untagging issue
when the vlan member is untagged and then it is changed to tagged the api is successful but the database is not getting updated.

### PR DESCRIPTION
**Pull request by rahul-1475**
Fixes #38 

- replaced openconfig url with sonic url to add vlan members.
- added new test case to check if_mode update.

### CODE REVIEW

File: network/vlan.py [hunk 1]
```diff
@@ -239,9 +239,7 @@ def vlan_mem_config(request):
                 ## Update members dicxtionary with tagging mode Enum
                 for mem_if, if_mode in members.items():
                     try:
-                        del_vlan_mem(
-                            device_ip, vlanid, mem_if, IFMode.get_enum_from_str(if_mode)
-                        )
+                        del_vlan_mem(device_ip, vlan_name, mem_if)
```
Comment (kamalkrbh) [inline network/vlan.py:242]: @rahul-1475 , vlanid at line 231 is not used.

File: network/test/test_vlan.py [hunk 2]
```diff
@@ -292,3 +291,77 @@ def get_req_body(self):
...
+            response.json()["mem_ifs"][self.portchnl_1],
+            str(IFMode.get_enum_from_str(request_body["mem_ifs"][self.portchnl_1])),
+        )
+        self.assertEqual(
+            response.json()["mem_ifs"][self.portchnl_2],
+            str(IFMode.get_enum_from_str(request_body["mem_ifs"][self.portchnl_2])),
+        )
+
+        #Cleanup
+        self.cleanup_vlan_mem_and_config(request_body)
```
Comment (kamalkrbh) [inline network/test/test_vlan.py:367]: @rahul-1475 , Assert the cleanup as well to insure vlan and member config is actually cleaned.

Comment (rahul-1475) [reply network/test/test_vlan.py:367]: cleanup_vlan_mem_and_config method vlaidates both members and vlan are cleaned up

Comment (kamalkrbh) [reply network/test/test_vlan.py:367]: OK Thanks.

File: network/vlan.py (hunk 1 above)
Comment (rahul-1475) [reply network/vlan.py:242]: removed vlan id validation from line 231 and 94

**Comment by rahul-1475** [comment]
other pr's

https://github.com/STORDIS/orca_nw_lib/pull/69

**Pull request merged**